.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks":{
+    "pre-push": "lint-staged && yarn test"
+  }
+}
```

Comment:
pre-push hook to lint/pretter, then run unit testing. Can be skipped with --no-verify ( and then caught in CI )